usr/share/rear/init/default/01_set_drlm_env.sh

```diff
@@ -6,8 +6,7 @@ fi
 
 PROGS=( "${PROGS[@]}" curl )
 
-drlm_import_runtime_config
+# Needed for curl with NSS support
+LIBS=( "${LIBS[@]}" /usr/lib64/libsoftokn3.so /usr/lib64/libsqlite3.so.0 /lib64/libfreeblpriv3.so )
```

Comment:
This code will not work for all platforms. If you want to have it in the default/ part then please modify it to be more generic.

Also, on my Ubuntu 14.04 system curl needs (according to `ldd`) only the libsqlite3.so.0, not the others. Are you sure that they are loaded on demain via dl_open()?

Did you really find out through experimentation that this is required?